Project-Music-Data/index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script type="module" src="script.js"></script>
```

Comment:
Interesting, I was expecting this to just be a `text/javascript` file rather than a module, but module works too!

Reply:
Not necessarily the expert here, but I'm fairly sure that to be able to it has to be `module` - since `import`s are an EcmaScript _Module_ feature.

Reply:
I was assuming functions would just get added to global scope, like the bad old days 😂